src/index.ts

```diff
@@ -9,7 +9,7 @@ export const NATO_PHONETIC_ALPHABET: PhoneticAlphabet = {
   f: 'foxtrot',
   g: 'golf',
   h: 'hotel',
-  i: 'indiana',
+  i: 'india',
```

Comment:
Could you update the snapshot test?

```sh
npm test -- -u
```

Reply:
Thanks for all of the suggestions, I have updated the commit message and updated the snapshot test with your instructions

Reply:
Oh, I forgot to check in the snapshot, just added another revision